.github/workflows/release.yml

```diff
@@ -22,13 +25,33 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - name: Create draft release
+        if: github.ref_type != 'branch'
         run: |
           gh release create \
             --repo ${{ github.repository }} \
             --title ${{ github.ref_name }} \
             --notes '' \
             --draft \
             ${{ github.ref_name }}
+
+      - uses: actions/checkout@v4
+        if: github.ref_type == 'branch'
+        with:
+          fetch-depth: 50
+
+      - name: Update ${{ github.ref_name }}-latest
+        if: github.ref_type == 'branch'
+        run: |
+          ref_name=${{ github.ref_name }}-latest
+          git tag $ref_name --force
+          git push origin $ref_name --force
+          gh release create \
+            $ref_name \
+            —-latest=false \
+            --title $ref_name \
+            --notes "Automated release for latest ${{ github.ref_name }}." \
+            || true
```

Comment:
We can't ignore this step, right? The prebuilt step requires the release to be presented.

Reply:
this particular `gh release create` will succeed only ever once per branch so I thought it's OK to ignore errors. I could make it more explicit or at least add a comment though.